Hazel/src/Hazel/Application.cpp

```diff
@@ -78,4 +78,23 @@ namespace Hazel {
 		return true;
 	}
 
-}
\ No newline at end of file
+	std::string Application::CorrectFilePath(const std::string& path)
+	{
+		#if defined(HZ_DEBUG) || defined(HZ_RELEASE)
+			struct stat buffer;
+			int status;
+
+			std::vector<std::string>prepath_string_vector = {"", "./Sandbox/", "../../../Sandbox/"};
```

Comment:
Doesn't look like a great idea to me to hard-code the name/directory of the application in the engine itself.
If we made a game with Hazel then we would have to change this line of engine code in order to run the game.

Reply:
That's right. Got me. Hmmm.